src/Npgsql.GeoJSON/GeoJSONHandler.cs

```diff
@@ -41,14 +41,17 @@ public override NpgsqlTypeHandler<GeoJSONObject> Create(PostgresType postgresTyp
                              "SELECT min(srid), max(srid), auth_name " +
                              "FROM(SELECT srid, auth_name, srid - rank() OVER(ORDER BY srid) AS range " +
                              "FROM spatial_ref_sys) AS s GROUP BY range, auth_name ORDER BY 1;", conn))
-                     using (var reader = cmd.ExecuteReader())
+                     {
+                         using var reader = cmd.ExecuteReader();
                          while (reader.Read())
                          {
                              builder.Add(new CrsMapEntry(
                                  reader.GetInt32(0),
                                  reader.GetInt32(1),
                                  reader.GetString(2)));
                          }
+                     }
```

Comment:
Don't trust Rider. It makes crappy things like this.

Reply:
What's crappy about this?

Reply:
In this method the body could be rewritten as:

```csharp
var builder = new CrsMapBuilder();
using var cmd = new NpgsqlCommand(
    "SELECT min(srid), max(srid), auth_name " +
    "FROM(SELECT srid, auth_name, srid - rank() OVER(ORDER BY srid) AS range " +
    "FROM spatial_ref_sys) AS s GROUP BY range, auth_name ORDER BY 1;", conn);
using var reader = cmd.ExecuteReader();

while (reader.Read())
{
    builder.Add(new CrsMapEntry(
         reader.GetInt32(0),
         reader.GetInt32(1),
         reader.GetString(2)));
}

return builder.Build();
```

Or it could be left as is so all `using`s are grouped and placed nearby.

Reply:
That's not quite the same. The reason Rider isn't rewriting it like you want, is because it would change the lifetime of `cmd`. If I'm reading the original code correctly, `cmd` was disposed before build.Build was invoked, and Rider correctly keeps it that way (otherwise it could produce some *very* broken code).

Reply:
Yep, so Rider did his job from the point of execution flow, but not from readability. Mixing two styles is what I dislike, and what Rider did. Therefore, let's go over changes and fix them.

Reply:
Well, you can't expect Rider to do anything else, since it would break programs.

What I propose, to move ahead and not get stuck, is to do a quick pass over this PR and merge it, and then we can always do a 2nd pass and use our judgment to correct on a case-by-case basis, what do you think? The vast majority of changes Rider did here are what we want, it seems.